go-controller/pkg/ovn/pod_selector_address_set.go

```diff
@@ -731,3 +739,31 @@ func deleteAddrSetsWithoutACLRef(predicateIDs *libovsdbops.DbObjectIDs,
 	}
 	return nil
 }
+
+func (bnc *BaseNetworkController) syncAddressSets() error {
```

Comment:
If this is only used by the default network controller, it should be moved to `default_network_controller.go`

Reply:
It would be needed for secondary networks as well for downstream.

Reply:
I think it should work with secondary networks with minor changes (just search for no-owner object and external-ids:network_name belongs to a certain network.